uportal-war/src/main/resources/layout/theme/respondr/content.xsl

```diff
@@ -411,7 +411,8 @@
       </xsl:if>
 
       <!-- Remove Icon -->
-      <xsl:if test="@unremovable='false' and not(//focused) and /layout/navigation/tab[@activeTab='true']/@immutable='false'">
+      <!-- note: deleteAllowed will either be false or not present if set from the admin ui -->
+      <xsl:if test="(@unremovable='false' and not(@dlm:deleteAllowed='false')) and not(//focused) and /layout/navigation/tab[@activeTab='true']/@immutable='false'">
```

Comment:
I can't recall any meaning that the 'unremovable' attribute has had in recent uPortal history.  I don't believe it was read anywhere in Universality, and I don't think there's any way to manage it in the UI.

If that's the case, I'd be happier simply to ignore it here, to prevent confusion.